### PR TITLE
Added platform `browser` support

### PR DESCRIPTION
**Pull request by mark-veenstra**
Now you can use the plugin to support the platform browser also to get the same version information

### CODE REVIEW

**Comment by mark-veenstra** [comment]
Any idea when and if you will merge this PR?

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.idea/
```
Comment (richsage) [inline .gitignore]: This should in your global `.gitignore`

Comment (georgeslegros) [inline .gitignore]: If this is the only blocker, you should definitely kill that line...

**Comment by richsage** [comment]
If this is still required, please amend and reopen against the `master` branch.

**Pull request closed without merging**